pages/IntenseDashboard.qml

```diff
@@ -192,6 +188,7 @@ Rectangle {
 
                 // try to start proxy and show error if it does not start
                 var haproxyStarted = callhaproxy.haproxy( walletHaproxyPath, Config.haproxyIp, Config.haproxyPort, endpoint, port.slice( 0,-4 ), 'haproxy', hexC( obj.id ).toString(), obj.provider, obj.providerName, obj.name )
+
                 if ( !haproxyStarted ) {
                     showProxyStartupError();
```

Comment:
The logic to abort payment needs to be in place anywhere showProxyStartupError() is called. I am thinking this is one of the primary areas responsible for still sending payment after failure.

Reply:
resolved in 386fcf4d10d35f800c0b4f3fadaf8335e63824b8